feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -0,0 +1,192 @@
+# AFT-2.1: AFTs Prefix Counters
+
+## Summary
+
+IPv4/IPv6 prefix counters
+
+## Procedure
+
+Configure DUT:port1 for IS-IS session with ATE:port1
+*   IS-IS must be level 2 only with wide metric.
+*   IS-IS must be point to point.
+*   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port1 to DUT:port1.
+*   IS-IS must be up and running, prefixes must be in RIB and FIB.
+
+Establish eBGP sessions between ATE:port1 and DUT:port1.
+*   Configure eBGP over the interface ip.
+*   Advertise 1000 ipv4,ipv6 prefixes from ATE port1 observe received prefixes at DUT.
+*   Validate total number of entries of AFT for IPv4 and IPv6.
+*   Each prefix must have 1 next hop pointing to ATE port1.
+
+Verifications
+
+*   BGP routes advertised from ATE:port1 must have 1 nexthop.
+*   IS-IS routes advertised from ATE:port1 must have one next hop.
+
+
+## AFT-2.1.1: AFT Prefix Counters ipv4 packets forwarded, IS-IS route.
+
+From ATE:port2 send 10000 pps to one of the ipv4 prefix advertise by IS-IS, after 1 minute stop the traffic.
```

Comment:
Potentially, we can replace this "1 minute of traffic" with a certain number of packets sent (e.g. send 100k, etc.)? This can save time in the test and make it a bit more consistent?

Reply:
ok